src/searchengine/nova/nova2dl.py

```diff
@@ -44,7 +44,7 @@
     try:
         exec("from engines.%s import %s" % (e, e))
         exec("engine_url = %s.url" % e)
-        supported_engines[engine_url] = e
+        supported_engines[engine_url] = e  # noqa: F821
```

Comment:
The code should not be aware of the linter, just remove F821 from the linter options.

Reply:
Replace with __global engine_url__ to help humans and linters understand the effect of the previous line (which is an attack vector).

Reply:
Replace with __global engine_url__ to help humans and linters understand the effect of the previous line (which is an attack vector).